.github/workflows/snapit.yml

```diff
@@ -0,0 +1,36 @@
+name: Snapit
+
+on:
+  issue_comment:
+    types:
+      - created
+
+env:
+  PNPM_VERSION: '8.7.0'
+
+jobs:
+  snapit:
+    name: Snapit
```

Comment:
Should we filter to [run only on PRs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment-on-issues-only-or-pull-requests-only)?

```suggestion
    name: Snapit
    if: ${{ github.event.issue.pull_request }}
```

Reply:
um, in theory it only runs for the specific comments, it shouldn't run on anything that's not a pull request no?

Reply:
Yeah I guess it doesn't matter. The action will run anyway because this `if` doesn't affect the event 🫤